common/static/common/js/components/views/feedback.js

```diff
@@ -4,7 +4,8 @@
             "underscore",
             "underscore.string",
             "backbone",
-            "text!common/templates/components/system-feedback.underscore"],
+            "text!common/templates/components/system-feedback.underscore",
+            "jquery.ui.selectable"],
```

Comment:
Here are the notes on what changed in version 11:

https://jqueryui.com/upgrade-guide/1.11/